docs/topics/maven-arguments.adoc

```diff
@@ -12,6 +12,13 @@ The following is a detailed description of the available {ProductShortName} {Mav
 [cols="40%,60%a",options="header"]
 |====
 |Argument |Description
+|analyzeKnownLibraries | Analyze known software artifacts embedded within your application. By default {ProductShortName} only analyzes application code.
+
```

Comment:
@RichardHoch  
Same as above - why isn't it a flag? And a comma after "By default"